Marlin/Configuration_adv.h

```diff
@@ -613,7 +637,9 @@
   // Use the amplification factor to de-/increase correction step.
   // In case the stepper (spindle) position is further out than the test point
   // Use a value > 1. NOTE: This may cause instability
+  // Must be 1.0 when using Z_STEPPER_ALIGN_KNOWN_STEPPER_POSITIONS
```

Comment:
Put this inside an #if DISABLED( block then in conditionals post use #if ENABLED on Z_STEPPER_ALIGN_KNOWN_STEPPER_POSITIONS and force it to 1.0